spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -402,14 +404,16 @@ def not_ok?; false; end
               if order.current == :second
                 :second_access
               else
-                order.pass_to :second, :resume_on => :sleep
```

Comment:
With your change will we start giving this new "probable deadlock" error for non-deadlock situations where threads where interleaved? That seems problematic and like it may break people's specs.